variables.es

```diff
@@ -0,0 +1,35 @@
+import pkg from "./package.json"
+
+// shell has some limitation :)
+const __PROD__ = JSON.parse(Boolean(process.env.__PROD__))
+
+const variables = {
+  __VERSION__: process.env.__VERSION__ || pkg.version,
```

Comment:
because this https://github.com/putaindecode/putaindecode.fr/blob/master/package.json#L71 allow to get latest commit hash as "version". And since this project is not a lib, it make more sense to me.